src/pages/home/ReportScreen.js

```diff
@@ -274,19 +285,46 @@ function ReportScreen({
             firstRenderRef.current = false;
             return;
         }
+
+        const onyxReportID = report.reportID;
+        const prevOnyxReportID = prevReport.reportID;
+        const routeReportID = getReportID(route);
+
+        // navigate to concierge when the room removed from another device (e.g. user leaving a room)
+        if (
+            // non-optimistic case
```

Comment:
```suggestion
```

Reply:
I think this tiny comment is required to explain why we have two separate logic

Reply:
I think it's fine without, but I won't block on it